docs/src/components/MarkdownPage/AdditionalLinks.js

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
This component handles all the extra content we add, like npm, github, edit page, cocoapod, etc links